src/hotspot/share/opto/superword.cpp

```diff
@@ -2359,472 +2052,6 @@ void SuperWord::schedule_reorder_memops(Node_List &memops_schedule) {
-        Node* mem = first->in(MemNode::Memory);
-        // Set the memory dependency of the LoadVector as early as possible.
-        // Walk up the memory chain, and ignore any StoreVector that provably
-        // does not have any memory dependency.
-        while (mem->is_StoreVector()) {
-          VPointer p_store(mem->as_Mem(), _vloop);
-          if (p_store.overlap_possible_with_any_in(p)) {
-            break;
-          } else {
-            mem = mem->in(MemNode::Memory);
-          }
-        }
-        Node* adr = first->in(MemNode::Address);
-        const TypePtr* atyp = n->adr_type();
-        vn = LoadVectorNode::make(opc, ctl, mem, adr, atyp, vlen, velt_basic_type(n), control_dependency(p));
-        vlen_in_bytes = vn->as_LoadVector()->memory_size();
-      } else if (n->is_Store()) {
-        // Promote value to be stored to vector
-        Node* val = vector_opd(p, MemNode::ValueIn);
-        if (val == nullptr) {
-          assert(false, "input to vector store was not created");
-          C->record_failure(C2Compiler::retry_no_superword());
-          return false; // bailout
-        }
-
-        Node* ctl = n->in(MemNode::Control);
-        Node* mem = first->in(MemNode::Memory);
-        Node* adr = first->in(MemNode::Address);
-        const TypePtr* atyp = n->adr_type();
-        vn = StoreVectorNode::make(opc, ctl, mem, adr, atyp, val, vlen);
-        vlen_in_bytes = vn->as_StoreVector()->memory_size();
-      } else if (VectorNode::is_scalar_rotate(n)) {
-        Node* in1 = vector_opd(p, 1);
-        Node* in2 = first->in(2);
-        // If rotation count is non-constant or greater than 8bit value create a vector.
-        if (!in2->is_Con() || !Matcher::supports_vector_constant_rotates(in2->get_int())) {
-          in2 =  vector_opd(p, 2);
-        }
-        vn = VectorNode::make(opc, in1, in2, vlen, velt_basic_type(n));
-        vlen_in_bytes = vn->as_Vector()->length_in_bytes();
-      } else if (VectorNode::is_roundopD(n)) {
-        Node* in1 = vector_opd(p, 1);
-        Node* in2 = first->in(2);
-        assert(in2->is_Con(), "Constant rounding mode expected.");
-        vn = VectorNode::make(opc, in1, in2, vlen, velt_basic_type(n));
-        vlen_in_bytes = vn->as_Vector()->length_in_bytes();
-      } else if (VectorNode::is_muladds2i(n)) {
-        assert(n->req() == 5u, "MulAddS2I should have 4 operands.");
-        Node* in1 = vector_opd(p, 1);
-        Node* in2 = vector_opd(p, 2);
-        vn = VectorNode::make(opc, in1, in2, vlen, velt_basic_type(n));
-        vlen_in_bytes = vn->as_Vector()->length_in_bytes();
-      } else if (opc == Op_SignumF || opc == Op_SignumD) {
-        assert(n->req() == 4, "four inputs expected");
-        Node* in = vector_opd(p, 1);
-        Node* zero = vector_opd(p, 2);
-        Node* one = vector_opd(p, 3);
-        vn = VectorNode::make(opc, in, zero, one, vlen, velt_basic_type(n));
-        vlen_in_bytes = vn->as_Vector()->length_in_bytes();
```

Comment:
Note: see `SuperWordVTransformBuilder::make_vtnode_for_pack` and `VTransformElementWiseVectorNode::apply`.